packages/gostringungarbler.vm/gostringungarbler.vm.nuspec

```
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>gostringungarbler.vm</id>
+    <version>1.0.0</version>
+    <authors>Chuong Dong</authors>
+    <description>GoStringUngarbler deobfuscates strings in Go binaries obfuscated by garble</description>
+    <dependencies>
+      <dependency id="common.vm" version="0.0.0.20240509" />
```

`VM-Get-Category` was introduced in (version `0.0.0.20250206` of common.vm`). We need to require that version or the package installation may fail.
```suggestion
      <dependency id="common.vm" version="0.0.0.20250206" />
```

---

I have also opened an issue to fix the scripts: https://github.com/mandiant/VM-Packages/issues/1369